cmd/admin-subnet-register.go

```diff
@@ -157,31 +157,27 @@ Please follow these steps to complete the registration:
 
 1) Copy the registration token shown above
 2) Open ` + subnetRegisterPageURL + `
-3) Paste the registration token there and submit the form
-4) Copy the license string generated
-5) Paste it here: `)
+3) Paste the registration token and submit the form
+4) Copy the API key generated
+5) And paste it here: `)
 
 	reader := bufio.NewReader(os.Stdin)
-	lic, e := reader.ReadString('\n')
-	fatalIf(probe.NewError(e), "Error in reading license string")
-	lic = strings.Trim(lic, "\n")
-
-	if len(lic) > 0 {
-		e := verifySubnetLicense(lic)
-		fatalIf(probe.NewError(e), "Invalid license specified:")
-		setSubnetLicenseConfig(alias, lic)
+	apiKey, e := reader.ReadString('\n')
+	fatalIf(probe.NewError(e), "Error in reading API key")
+	apiKey = strings.Trim(apiKey, "\n")
```

Comment:
```suggestion
	apiKey = strings.TrimSpace(apiKey)
```
This will trim `\r` on Windows as well as any spaces included in the copy+paste.

Reply:
Thanks!